### PR TITLE
update docs, add cli docker release

### PR DESCRIPTION
**Pull request by rolang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,33 +27,44 @@ Each versioned migration must be assigned a unique version.
...
 ## Usage example
 
+For usage via command line see [command-line](#command-line) section.
+
+In a sbt project dumbo can be added like:
+
+```
+libraryDependencies += "dev.rolang" %% "dumbo" % "0.0.7"
+```
+
```
Comment (greshny) [inline README.md:67]: I would also mention in documentation about snapshot releases that you need to configure `resolvers`, e.g.

```scala
ThisBuild / resolvers +=
  "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
```

Comment (rolang) [reply README.md:67]: right, will add, thanks for the hint :+1:

Comment (rolang) [reply README.md:67]: updated

Comment (greshny) [reply README.md:67]: thanks

**Pull request merged**